app/views/course/course_users/show.html.slim

```diff
@@ -0,0 +1,23 @@
+= page_header
+
+div.row
+  div.profile-picture.col-xs-4.col-sm-3.col-lg-2
+    = display_user_image(@course_user.user)
+
+  div.col-xs-8.col-sm-9.col-lg-10
+    h2 = format_inline_text(@course_user.name)
+    p = @course_user.user.email
+    p = t('.role', role: @course_user.role.titleize)
+
+    - if @course_user.student?
+      p = t('.achievement_count', count: @course_user.achievement_count)
+
+      - if can?(:read, Course::ExperiencePointsRecord.new(course_user: @course_user))
+        - path = course_course_user_experience_points_records_path(current_course, @course_user)
+        - points_with_link = link_to @course_user.experience_points, path
```

Comment:
can't the entire line be a link instead? then you won't need to construct the link like this.